docs/commands.md

```diff
@@ -152,6 +152,7 @@ An array with an entry per execution. Each entry is made of alternating key name
 
 * **executionId**: the [execution ID](functions.md#execution-id)
 * **status**: the [status](functions.md#execution-status)
+* **registered execution**: Indication that the given execution is a registered execution plan.
```

Comment:
An indication that the given execution was triggered by a registration?

@itamarhaber what do you think?

Reply:
1. I don't like the field name having a space and being so long, would replace it with `registered`
2. (note the lowercase i and no ending period)
```suggestion
* **registered**: indicates whether this is a [registered execution](functions.html#registration)
```